### DISCUSSION TITLE
Grafana dashboard for this tool

### DISCUSSION THREAD

**Discussion by idan3** [trallnag/prometheus-fastapi-instrumentator#318] [Q&A]
Is there an out-of-the-box Grafana dashboard fit for this tool?

**Comment by jpizquierdo** [comment]
that would be awesome! maybe someone can share his/her dashboard with us :)

**Reply by jpizquierdo** [reply]
@idan3 indeed I found @Kludex implementation using this repo. He has build a grafana visualization out of the box which is great:
https://github.com/Kludex/fastapi-prometheus-grafana
